views/components/dropdowns/teams.jsx

```diff
@@ -16,11 +16,6 @@ module.exports = React.createClass({
   },
   render: function () {
     var dropdown;
-    var attributes = {};
```

Comment:
Why did you removed this part of the code?
It handles the disabling of the teams buttons when no Organization is selected.